AucCar/portal/admin.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib import admin
+from .models import *
+
+# Register your models here.
+
+class DurationAdmin(admin.ModelAdmin):
+	list_display = ('duration_name','duration_start_date','duration_end_date',)
+
+
+class PersonTypeAdmin(admin.ModelAdmin):
+	list_display = ('person_type','person_sub_type',)
+
+class CurrencyAdmin(admin.ModelAdmin):
+	list_display = ('currency_name',)
+
+class RegistrationFairAdmin(admin.ModelAdmin):
+	list_display = ('registration_fair_name','registration_fair_amount','get_currency',)
+	def get_currency(self, obj):
+		return obj.currency.currency_symbol
+
+class RegistrationStatusAdmin(admin.ModelAdmin):
+	list_display = ('registration_status_name',)
+
+class RegistrationAdmin(admin.ModelAdmin):
+	list_display = ('person_name','get_per_type','get_reg_fair','get_status',)
+	def get_reg_fair(self, obj):
+		return obj.registration_fair.registration_fair_name
+	def get_per_type(self, obj):
+		return obj.person_type.person_type
+	def get_status(self, obj):
+		return obj.registration_status.registration_status_name
+
```

Comment:
```suggestion
search_fields = ['person_uid']
``` add this line to enable search @Amr2012Hashem